game-api/src/games/test.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+const path = require("path");
+const stripAnsi = require("strip-ansi");
+const docker = new (require("dockerode"))();
+
+const { gameDir, debugLog } = require("../cliArgs");
+const CommonDockerGameManager = require("./common-docker-game-manager");
+
+module.exports = class TestManager extends CommonDockerGameManager {
```

Comment:
This should probably just be an "interface"/sample by making the methods just return true/1/etc rather than copypasta'ing the Factorio one (and shouldn't extend CommonDockerGameManager).